radixdlt/src/main/java/com/radixdlt/store/SearchCursor.java

```diff
@@ -1,23 +1,23 @@
-/*
- * (C) Copyright 2020 Radix DLT Ltd
- *
- * Radix DLT Ltd licenses this file to you under the Apache License,
- * Version 2.0 (the "License"); you may not use this file except in
- * compliance with the License.  You may obtain a copy of the
- * License at
- *
- *  http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
- * either express or implied.  See the License for the specific
- * language governing permissions and limitations under the License.
- */
-
+/*

```

Comment:
Spurious diff

Reply:
Please look at the other spurious diffs, e.g. the ones where the copyright notice changed in diff but the content actually remained the same. Usually due to a formatting issue, would like to keep these unnecessary changes to a minimum for better traceability